content/md/franklin-insights/franklin-insights-subscriptions.md

```diff
@@ -16,16 +16,33 @@ When you subscribe a product to Franklin, Franklin **collects and validates info
 
 You have 2 ways to subscribe products to Franklin:
 
-- Via the **Product Edit Form**
+## Via the **Product Edit Form**
 
 ![image](../img/Akeneo_Screenshot_Franklin_Insights_PEF_Subscription.png)
 
-- Via **bulk action**
+## Via **bulk action**
 
 ![image](../img/Akeneo_Screenshot_Franklin_Insights_Bulk_Actions.png)
 
 In order for you to receive attribute values proposals for subscribed and enriched products, you need to **match between Franklin’s attributes and your catalog attributes** in each family using the [Franklin attributes mapping UI](franklin-insights-mappings.html#identifiers-mapping).
 
+# Filter on subscribed products in the product grid (_since 3.1_)
+
+In order to easily find the subscribed or unsubscribed products, we added a filter `Franklin subscription` in the product grid. Thus, you can easily display the products in the grid and [bulk subscribe or unsubscribe] them.
```

Comment:
Why the brackets? "[bulk subscribe or unsubscribe]"

Reply:
I wanted to add a link to an article and forgot it. Fix in progress

Reply:
```suggestion
In order to easily find the subscribed or unsubscribed products, we added a filter `Franklin subscription` in the product grid. Thus, you can easily display the products in the grid and [bulk subscribe or unsubscribe](franklin-insights-subscriptions.html#via-bulk-action) them.
```